### ISSUE TITLE
Accessibility: Add ability to tab to bookmarks and support/oppose

### ISSUE THREAD

**Issue by DaleMcGrew** [wevote/WebApp#227]
1. Go to the ballot
2. Tab through to the page and you find that you skip past bookmarks and support/oppose
   A person should be able to use the keyboard to get to all interface elements.

**Comment by wendy-c** [comment]
works on chrome, still skipping support/oppose on safari

**Comment by zachmonteith** [comment]
update: hard coding in a tab index through dev tools does not cause firefox to highlight candidate name, so that theory is debunked

**Comment by ixley** [comment]
@DaleMcGrew This looks like it has been addressed. On a related note – tabbing typically only jumps to links and buttons, so text that has a click handler attached, but has no `href` will not work. On /ballot, notice that tabbing catches the 'office' text links but not the candidate ones.

**Comment by DaleMcGrew** [comment]
Hi @ixley, @nickelser is still working on this. We got it working on all browsers except Safari, and Nick tracked down a possible fix that he is working on.

**Comment by ixley** [comment]
ah okay - sorry about that.

On Sun, Apr 30, 2017 at 10:01 AM, Dale John McGrew <notifications@github.com

**Comment by nickelser** [comment]
the problem actually stems from the default safari behavior -- which doesn't match any of the other browsers.

this SO answer describes the solution: https://stackoverflow.com/a/1914496

we could notify safari users that they should change their settings, but i assume that anyone who has accessibility needs and uses safari would already have it enabled.

i recommend actually not changing any code or HTML and instead trusting the users 😄
